packages/sdk/src/evm/constants/erc1155-features.ts

```diff
@@ -24,40 +25,40 @@ export const FEATURE_EDITION_BURNABLE = {
 } as const;
 
 export const FEATURE_EDITION_CLAIM_CONDITIONS_V1 = {
-  name: "ERC1155ClaimConditionsV1",
+  name: "ERC1155DropSinglePhaseV1",
```

Comment:
on thing i realized is that these names are used for autogenerating docs.

You need to look for ALL occurrences of this in this repo, but also in python and go :| 

You also need to look for occurences of these in the dashboard repo, because we rely on this string for contract detection.

and i'm not 100% sure what the docs repo relies on in terms of side bar, but it probably impacts it too.